src/overlays/actors/ovl_En_Gakufu/z_en_gakufu.c

```diff
@@ -115,9 +115,12 @@ void EnGakufu_ProcessNotes(EnGakufu* this) {
 
     songIndex = this->songIndex;
     ocarinaSongButtons = &gOcarinaSongButtons[songIndex];
+
+    //! FAKE:
+    if (1) {}
     songNumButtons = gOcarinaSongButtons[this->songIndex].numButtons;
 
-    for (i = 0; i < songNumButtons; i++) {
+    for (i = 0; i < (u8)songNumButtons; i++) {
```

Comment:
What happened here?

Reply:
This was the result of changing:
`u32 AudioOcarina_SetInstrument(u8 ocarinaInstrumentId);`
to
`void AudioOcarina_SetInstrument(u8 ocarinaInstrumentId);`

The latter should be the correct one, and indeed `void` is required to match these quest functions. However, changing that function to `u32` was a trick to match that gakufu function, but it seemed like a fake match to begin with. An alternative match was found by Anon (huge thanks to them), but presumably there's something better out there. Here's the conversation where I mention it:

https://discord.com/channels/688807550715560050/688851997352263820/1013314490625949718